common/v2/components/WalletUnlock/DeterministicWallets.tsx

```diff
@@ -26,6 +28,18 @@ import { DeterministicWalletData, getDeterministicWallets } from 'v2/services/Wa
 import { getBaseAssetBalances, BalanceMap } from 'v2/services/Store/BalanceService';
 import Account from '../Account';
 
+const FooterLink = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: baseline;
+  p {
+    font-size: 16px;
+  }
+  a {
+    color: ${COLORS.BLUE_BRIGHT};
```

Comment:
When using the translation you can use Markdown, so we don't have to do the extra styling here.

Reply:
Nice tip !

Reply:
But you can't change color with MD ? 🤔

Reply:
It'll use standard styling for links, like we use on the rest of the site.